app/src/main/res/values/strings.xml

```diff
@@ -593,4 +593,7 @@ Upload your first media by tapping on the add button.</string>
   <string name="upload_nearby_place_found_description">Is this a photo of Place %1$s?</string>
   <string name="remove_bookmark">Removed from bookmarks</string>
   <string name="add_bookmark">Added to bookmarks</string>
+  <string name="wallpaper_set_unsuccessfully">Error setting wallpaper</string>
```

Comment:
Just a suggestion, thus could be "Something went wrog..could not set the wallpaper"

Reply:
@ashishkumar0207 Thanks I will make the change.